.gitignore

```diff
@@ -38,7 +38,6 @@ bundle/
 publish_to_bower/
```

Comment:
- [ ] Double check that entries aren't preventing us from checking in useful files in `packages/pds-ember`.

Reply:
Cleaned up old entries and kept what applied, at present. We can update entries as needed along the way.